### PR TITLE
Promotion of Pension Tool / Moving Book appointment links

### PR DESCRIPTION
**Pull request by Guntrisoft**
Added link to the pension type tool to all options pages and pension statements page to potentially increase click-through rate and conversion. Moved book appointment link from under next steps section of options pages to within body copy of pages to provide greater visibility.

### CODE REVIEW

File: content/pension_statements.md [hunk 1]
```diff
@@ -6,30 +6,33 @@ description: Your pension statement shows your pension type, how much is in your
...
 You normally get one pension statement a year – your provider may call this your annual or yearly statement.
 
-Your statement shows how much is in your pot and usually an estimate of how much you might get when you start taking money from it.
+Your statement shows:
 
-^Knowing what’s in your pension statement will help when you come to [book a free appointment](/appointments) with a Pension Wise guidance specialist.^
+ - how much is in your pot
+ - an estimate of how much you might get when you start taking your money
+ - if your pension has any special features, eg guaranteed annuity rate
+ - your 'selected retirement age' (the age you agreed with your provider to retire)
```
Comment (benbarnett) [inline content/pension_statements.md]: Should we be using the curly quotes here?

**Comment by Guntrisoft** [comment]
@alexkouz Alex can you tidy up the description of this PR, on why we are doing these content changes

File: content/pension_statements.md (hunk 1 above)
Comment (alexkouz) [inline content/pension_statements.md]: Yes, I believe reuse curly ones. Will amend.

Comment (alexkouz) [inline content/pension_statements.md]: Okay, file now amended.

**Comment by benlovell** [comment]
[Add links to pension type tool across site](https://trello.com/c/xW4fKIxt/20-add-links-to-pension-type-tool-across-site)

**Pull request merged**